### PR TITLE
Fix Q061

### LINKED ISSUE #1076: Q061: incorrect field reference
Q061 is coded to check the proportion of HOEPA status but should instead be checking property type.

### PR DESCRIPTION
**Pull request by nickgrippin**
Closes #1076

### CODE REVIEW

File: validation/src/main/scala/hmda/validation/rules/lar/macro/Q061.scala [hunk 1]
```diff
@@ -20,14 +20,14 @@ object Q061 extends AggregateEditCheck[LoanApplicationRegisterSource, LoanApplic
 
   override def apply[as: AS, mat: MAT, ec: EC](lars: LoanApplicationRegisterSource): Future[Result] = {
 
-    val firstLienHoepaLoans =
-      count(lars.filter(lar => lar.hoepaStatus == 1 && lar.actionTakenType == 1 && lar.lienStatus == 1 && lar.rateSpread != "NA")
+    val firstLienPropertyLoans =
+      count(lars.filter(lar => lar.loan.propertyType == 1 && lar.actionTakenType == 1 && lar.lienStatus == 1 && lar.rateSpread != "NA")
         .filter(lar => lar.rateSpread.toInt >= 5))
```
Comment (schbetsy) [inline validation/src/main/scala/hmda/validation/rules/lar/macro/Q061.scala]: (I know this isn't the stated issue, but it's worth looking into while we're here) Should this rateSpread comparison really use `toInt`? Will that cause rounding issues?

**Comment by nickgrippin** [comment]
@schbetsy: Good catch, definitely could cause problems.  Also noticed that it should be `>`, not `>=`.  Done in 117f80e903e2a5a7768920be04006f5b601bd9b8

**Comment by schbetsy** [comment]
I submitted the Q061 test file (in the shared drive at HMDA > Platform > HMDA Edits > Validation Testing > Test Files > Q061.txt) to this updated branch and `Q061` did fail. Is that the desired outcome, @Kibrael?

**Comment by schbetsy** [comment]
@nickgrippin I saw this test failure in Travis:
```
[info] Q061Spec:
[info] - be invalid if first lien property type loans > 0.01 * total *** FAILED ***
[info]   Success() was not an instance of hmda.validation.dsl.Failure, but an instance of hmda.validation.dsl.Success (SummaryEditCheckSpec.scala:26)
```

Maybe there's an edge case not accounted for in the specs?

**Comment by nickgrippin** [comment]
@schbetsy: It wasn't an edge case, I forgot to update the tests with the new "greater than" logic.  Should be fixed now.

**Comment by schbetsy** [comment]
Excellent. And @Kibrael confirmed that the test file should indeed fail, so this is showing the correct behavior now. LGTM, once Travis is green.

**Comment by Kibrael** [comment]
+1
